pmdarima/utils/visualization.py

```diff
@@ -371,6 +372,15 @@ def tsdisplay(y, lag_max=50, figsize=(8, 6), title=None, bins=25,
     ax1 = fig.add_subplot(gs[2:, 0])
     ax2 = fig.add_subplot(gs[2:, 1])
 
+    # make sure y is a np array
+    y = check_endog(y, copy=False)
+
+    if lag_max >= y.shape[0]:
+        raise ValueError(
+            f"lag_max ({lag_max}) must be < than length of the "
```

Comment:
Nit: `<` means "less than," so this reads as "lag_max (x) must be less than *than* length...". I think we should remove one or the other. I would do "less than" (with no symbol), but up to you

Reply:
If only there were an eyeroll emoji. 😉

Reply:
Like this? 🙄

Reply:
https://media.tenor.com/images/f8a02c67648240f3eba5b3fb871e7c37/tenor.gif

Reply:
![pepe reeeee](https://media.tenor.com/images/f8a02c67648240f3eba5b3fb871e7c37/tenor.gif)

:trollface: